### PR TITLE
Add steps for PR release's assignee

### PR DESCRIPTION
**Pull request by benjben**

### CODE REVIEW

File: release-process.md [hunk 1]
```diff
@@ -103,6 +103,14 @@ git push --tags
 5. Only after you're sure that CI/CD was successful and asset is published, you can push the `master` branch: `git push origin master`. This will automatically close all tickets in the milestone. By pushing `master` branch last you're making sure that you won't have to force-push to `master` late because of broken CI/CD
 6. Push the release branch. It should automatically close the associated PR and mark it as "merged".
 
+### Release: PR's assignee
+
+1. If the code has not been reviewed yet, review it.
+1. Check that there is a 1:1:1 relationship between commit log, changelog file and issue tracker.
```
Comment (chuwy) [inline release-process.md]: All indexes are `1`.

File: release-process.md [hunk 2]
```diff
@@ -103,6 +103,14 @@ git push --tags
 5. Only after you're sure that CI/CD was successful and asset is published, you can push the `master` branch: `git push origin master`. This will automatically close all tickets in the milestone. By pushing `master` branch last you're making sure that you won't have to force-push to `master` late because of broken CI/CD
 6. Push the release branch. It should automatically close the associated PR and mark it as "merged".
 
+### Release: PR's assignee
```
Comment (chuwy) [inline release-process.md]: I think these things can be checked by authors themselves? I'd rename it to something like `Final checklist` - these are things we forget most often.

Comment (chuwy) [reply release-process.md]: As of reviews - I'd add that we must have at least two sign offs before releasing.

File: release-process.md [hunk 3]
```diff
@@ -103,6 +103,14 @@ git push --tags
...
+
+1. If the code has not been reviewed yet, review it.
+1. Check that there is a 1:1:1 relationship between commit log, changelog file and issue tracker.
+1. Check that each component that has been updated in the PR has its version bumped, and that the commit for the bump is after the commits that updated it.
+1. In `Prepare for release commit`:
+    - In the `README.md` Check that `[release-image]` is updated with the new release.
+    - Chech that `VERSION` is updated.
```
Comment (chuwy) [inline release-process.md]: ...if `VERSION` is present. We have it only in monorepos such as snowplow/snowplow, snowplow/snowplow-rdb-loader etc.

File: release-process.md (hunk 1 above)
Comment (benjben) [reply release-process.md]: Github takes care of creating the right numbers when displaying, this way you don't need to manually update all the indexes when you add/delete one line.

File: release-process.md (hunk 2 above)
Comment (benjben) [reply release-process.md]: Indeed they should! Then I think that there is not really a checklist for the PR's assignee, as he should only re do the same checks like the author of the PR. I'll rename it as you said :+1:

**Comment by dilyand** [comment]
This is awesome!

**Comment by benjben** [comment]
I updated the numbering @chuwy .

Given that 2 reviewers approved the PR, I'm merging.

**Pull request merged**